components/stratum/stratum_api.c

```diff
@@ -171,6 +181,11 @@ void STRATUM_V1_parse(StratumApiV1Message * message, const char * stratum_json)
                 message->response_success = true;
             } else {
                 message->response_success = false;
+                if (!cJSON_IsNull(reject_reason_json)) {
```

Comment:
Nit: Technically I think you don't need the IsNull, as both check for a null value and IsString will return false if the type is `cJSON_NULL`.